src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const BASE_COST = 40;
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    return days * BASE_COST - SHORT_TERM_DISCOUNT;
+  }
+
+  if (days >= LONG_TERM) {
+    return days * BASE_COST - LONG_TERM_DISCOUNT;
```

Comment:
Similar to the short-term discount, the long-term discount is applied as a fixed amount. Verify if this is the intended discount method, as a percentage discount might be more appropriate.